src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -983,4 +995,20 @@ export class QuizExerciseDetailComponent extends QuizExerciseValidationDirective
         }
         return super.computeInvalidReasons().concat(invalidReasons);
     }
+
+    getReleaseDateExplanation(quizMode?: QuizMode) {
+        switch (quizMode) {
+            case QuizMode.SYNCHRONIZED:
+                return 'artemisApp.quizExercise.releaseDateExplanation.synchronized';
+            case QuizMode.BATCHED:
+                return 'artemisApp.quizExercise.releaseDateExplanation.batched';
+            case QuizMode.INDIVIDUAL:
+                return 'artemisApp.quizExercise.releaseDateExplanation.individual';
+        }
+        return undefined;
```

Comment:
I would suggest to keep it as it is because this allows searching the translation string as a whole in the code which I think is helpful during development. It also allows seeing the unhandled QuizModes more clearly which can be useful if  new QuizMode is added but forgot to be adapted fully. What do you think?

Reply:
On one hand yes, being able to find all used string in the code would be nice if we were to ever do a cleanup of the existing translations but on the other hand this kind of pattern is used in several other places and there are even strings where parts come from both server and client. Additionally when adding new modes this wouldn't cause a compile error anyways it would just attempt to translation undefined instead of 'artemisApp.quizExercise.releaseDateExplanation.XXX'. Finally while this is rather minor compared to some of the other code duplication issues that plague the Artemis code base I don't think this is very useful.

Reply:
Makes sense, I made the change and removed the method altogether. I added the single line you suggested to the .html file.